site/_posts/2020-09-03-oidc-auth-api-usage.md

```diff
@@ -80,13 +85,39 @@ oidc_client_id="my-oidc-client"
 
 # OIDCOauthIntrospectionEndpoint from the OpenID-Connect configuration
 oidc_auth_introspection_endpoint="https://keycloak.example.com/auth/realms/miq/protocol/openid-connect/token/introspect"
+```
+
+  * The value of **oidc_provider_token_endpoint** varies based on the configuration and the OpenID-Connect Provider being used. 
+
+  * When using IBM's Identiy Access Management (IAM) for the OpenID-Connect Provider, the value for **oidc_provider_token_endpoint** will be the value of OIDCProviderTokenEndpoint from the OpenID-Connect configuration with **token** replace with **identitytoken**.
+
+  * **e.g.: Using IBM's Identiy Access Management (IAM) with the following OpenID-Connect configuration:**
+
+```bash
+OIDCProviderTokenEndpoint https://cp-console.apps.joevocplusmcmim.cp.fyre.ibm.com/idprovider/v1/auth/token
```

Comment:
I'd use a sample domain for upstream doc and not a real one. maybe iam.example.com.